SU2_PY/SU2/opt/scipy_tools.py

```diff
@@ -142,7 +142,241 @@ def scipy_slsqp(project,x0=None,xb=None,its=100,accu=1e-10,grads=True):
     # Done
     return outputs
     
-    
+# -------------------------------------------------------------------
```

Comment:
A have had a lot of problems with the constrained optimization, but the unconstrained methods are really great! You have probably noticed that the design variable bounds are inside the SU2 grid deformation subroutine... so you can run unconstrained methods with bounds in the design variables.